syntax = "proto3";

package lbs;

message LbsInfo
{
    string id = 1;
    int32 strength = 2;
}

message LbsList
{
    repeated LbsInfo info = 1;
    string imei = 2;
    string method = 3; //本次调用是lbs还是wifi
}

message LbsRec
{
    string bsid = 1;
    double lon = 2;
    double lat = 3;
    double accuracy = 4;
}
